src/pages/SamplePage/SamplePage.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import {
+  IonBackButton,
+  IonButtons,
+  IonContent,
+  IonHeader,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+} from "@ionic/react";
+import { useParams } from "react-router";
+import { paths } from "../../Router";
+
+interface SamplePageParams {
+  submissionId: string;
+  sampleIndex: string;
+}
+
+const SamplePage: React.FC = () => {
```

Comment:
I was going to say I had already implemented a `PlaceholderPage`... then I realized this is referring to a biosample. I don't know that I would be in favor of renaming this to `BiosamplePage`, but I did want to record my initial confusion. I do want the user-facing UI to use the word "Sample" (I think it's a more user-friendly word and they'll know what it's referring to).